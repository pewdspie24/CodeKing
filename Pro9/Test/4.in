10
878589 985777
270723 834513
412963 930498
355008 749922
26668 148568
822849 995634
499008 743734
739195 829463
54626 154365
351159 696719
