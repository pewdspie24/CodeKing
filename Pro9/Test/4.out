1726
14500
11652
10010
5631
3014
5599
1542
4555
9367
